src/chi-vue/src/components/transfer-list/TransferListActions.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="chi-transfer-list__actions">
+    <button class="chi-button -icon -flat" type="button">
+      <i class="chi-icon icon-chevron-right"></i>
+    </button>
+    <button class="chi-button -icon -flat" type="button">
+      <i class="chi-icon icon-chevron-left"></i>
+    </button>
```

Comment:
@pedrotaglialatela-lumen could we change these buttons to web components? Same as we have in TransferListColumn.vue line 8 

```suggestion
    <button class="chi-button -icon -flat" type="button">
      <i class="chi-icon icon-chevron-right"></i>
    </button>
    <button class="chi-button -icon -flat" type="button">
      <i class="chi-icon icon-chevron-left"></i>
    </button>
```

Reply:
@jllr sure!! Updating